### PR TITLE
Added insert_many functions for all remaining parameters

### PR DESCRIPTION
**Pull request by palash-t**
Added insert_many functions for accel, biometric, game, test, and emotion

Also added corresponding test functionality.

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -161,4 +161,174 @@ def many_gyros():
...
+		"test_id": uuid.uuid4(),
+		"description": "test3",
+		"patient_id": uuid.uuid4(),
+		"test_score": 50
+	},
+	{
+		"test_id": uuid.uuid4(),
+		"description": "test4",
+		"patient_id": uuid.uuid4(),
+		"test_score": 69
```
Comment (snelson44) [inline tests/conftest.py:289]: lol

File: tests/test_database.py [hunk 2]
```diff
@@ -339,6 +344,140 @@ def test_insert_many_gyros(database, many_gyros):
 			assert result['y'] == y
 			assert result['z'] == z
 
+def test_insert_many_accels(database, many_accels):
+
+	accel_id = many_accels[1]["accel_id"]
+	description = many_accels[1]["description"]
+	patient_id = many_accels[0]["patient_id"]
```
Comment (snelson44) [inline tests/test_database.py]: these should all be index 1 for all of the insert many tests. I made a copy and paste error on mine. If you can fix mine too that would be great.

**Review by snelson44 — CHANGES_REQUESTED** [review]
Just one change that needs made, which was my bad

File: tests/test_database.py (hunk 2 above)
Comment (palash-t) [reply tests/test_database.py]: Done.

**Pull request merged**